auth0/v3/management/rest.py

```diff
@@ -96,12 +97,19 @@ def _parse(self, response):
 
 
 class Response(object):
-    def __init__(self, status_code, content):
+    def __init__(self, status_code, content, headers):
         self._status_code = status_code
         self._content = content
+        self._headers = headers
 
     def content(self):
         if self._is_error():
+            if self._status_code == 429:
+                reset_at = int(self._headers.get('x-ratelimit-reset', '-1'))
```

Comment:
headers once parsed by the library are put into a [CaseInsensitiveDict](https://requests.readthedocs.io/en/master/_modules/requests/structures/). Also, headers are by spec case insensitive -> https://requests.readthedocs.io/en/master/user/quickstart/#response-headers